src/guides/v2.3/graphql/mutations/apply-coupon.md

```diff
@@ -114,3 +114,14 @@ Attribute |  Data Type | Description
 {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`A coupon is already applied to the cart. Please remove it to apply another` | Specified coupon in the `coupon_code` attribute is already applied to cart. Use [removeCouponFromCart]({{page.baseurl}}/graphql/mutations/remove-coupon.html) to remove a current one and to apply another coupon.
+`Cart does not contain products.` | Coupon can not be applied to an empty cart.
+`Could not find a cart with ID "XXX"` | The specified `cart_id` value does not exist in the `quote_id_mask` table.
+`Required parameter "coupon_code" is missing` | The required `coupon_code` attribute contains an empty value.
+`The coupon code isn't valid. Verify the code and try again.` | Coupon code has been not set.
```

Comment:
Hi @atwixfirster. Thank you for the detailed information. I would clarify this part. When we say "Coupon code has been not set", this explanation might not be very informative. I would rather say "The entered coupon code is not applicable. Check the existing shopping cart price rules for details."

Thank you

Thank you.